config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  namespace :api do
+    namespace :v1 do
+      resource :messages
+    end
+  end
```

Comment:
- [ ] Good project implementation :clap:  Unfortunately The React app can not use the url to your backend api to display  a random greeting as expected for this project because both back end and frontend app can not communicate because they are blocked by **CORS policy** as you can see on the screenshot below :arrow_down: Please fix it and make sure that your app display the random messages as expected :+1: 
![Screenshot_20230221_004448](https://user-images.githubusercontent.com/6615394/220221513-f3fdfc33-ecff-4be2-8a9d-102f41f93b6d.png)